zulip_bots/zulip_bots/bots/joinme/joinme.py

```diff
@@ -0,0 +1,102 @@
+import json
+import requests
+import socket
+from typing import Any, Dict, Text
+
+help_message = '''
+This is Joinme bot.
+* You can start meetings by @botname start meeting and then confirming by @botname confirm <authorization_code>.
+'''
+
+def get_start_meeting_response(message_content: str, config_info: Dict[str, str]) -> str:
+    Key = config_info['Key']
```

Comment:
Should this be lowercase?

Reply:
Actually, I tried it in lowercase also...